### DISCUSSION TITLE
Spectre Desktop connected to Raspberry Pi 4 node

### DISCUSSION THREAD

**Discussion by jdelgadoesteban** [cryptoadvance/specter-desktop#1117] [General]
Hello, I have my Spectre Desktop running with no issues on my Mac connected successfully to my local node running on a Raspberry Pi 4 with Umbrel. For some reasons, the funds of my Spectre Desktop are not showing on my node wallet. Has anyone experienced this issue?

Cheers

**Comment by ben-kaufman** [comment]
Hi, the Umbrel node wallet is completely separate and is just showing the one default wallet Umbrel created for you on setup. It doesn't show other wallets on your node.

**Reply by jdelgadoesteban** [reply]
@ben-kaufman Thanks for that, it is somehow counterintuitive as when you follow the process of selecting a wallet you are presenting with multiple options, Spectre native on Umbrel or connect to your Spectre Desktop. If the set up is as you mention I don´t really see any advantage in having a separate Spectre Desktop wallet on my mac, what do you think?

**Reply by ben-kaufman** [reply]
If you just want to use a hot wallet and that's it then yes there's no point, but when you want to use also hardware wallets then you'll need a software like Spceter

**Reply by jdelgadoesteban** [reply]
@ben-kaufman thanks for the clarification 🙏

**Reply by Rococolab** [reply]
Speaking of Spectre on Umbrel on Pi4. IS there a SAFE way to upgrade Spectre on Umbrel without having to wait for Umbrel to do the update. It's still on v.1.8.1
